client/me/purchases/controller.jsx

```diff
@@ -201,15 +216,15 @@ export default {
 			'Manage Purchase'
 		);
 
-		sites.setSelectedSite( context.params.site );
+		setSelectedSite( context.params.site, context.store.dispatch );
 
 		renderPage(
 			context,
-			<ManagePurchaseData
-				component={ ManagePurchase }
-				purchaseId={ context.params.purchaseId }
+			<ManagePurchase
+				purchaseId={ parseInt( context.params.purchaseId, 10 ) }
 				destinationType={ context.params.destinationType }
-				sites={ sites } />
+				user={ user.get() }
```

Comment:
Nice catch, I'll remove this from the props/propTypes. I'm not sure what I was thinking. :)